risedev.yml

```diff
@@ -848,6 +872,16 @@ template:
     # Jaeger has a lot of ports open, and we don't want to make this config more complex.
     # So we keep the default value of jaeger instead of making it part of RiseDev config.
 
+  # opendal:
+  opendal:
+    id: opendal
+
+    engine: hdfs
+
+    namenode: 127.0.0.1:9000"
+
+    root: risingwave
```

Comment:
Let's add some documentations for each field like other sections.

Reply:
Will integrate hdfs in risedev in next PR.

Reply:
May remove this or at least add a TODO here.